server/src/app.ts

```diff
@@ -128,7 +120,7 @@ smartSchedulingApi.use((_req: Request, res: Response) =>
 smartSchedulingApi.use(Sentry.Handlers.errorHandler());
 smartSchedulingApi.use(
   (error: any, req: Request, res: Response, _next: NextFunction) => {
-    console.error("ERRROR:", error);
+    logger.error("ERRROR:", error);
```

Comment:
I’m not sure it really makes sense to include the `ERROR` text here since the format you set up already prefixes this with the level. It doesn’t really seem useful to log `error: ERROR: {error}`.

This is also not really correct — `console.*()` takes N arguments that are formatted as appropriate for their type (e.g. strings are text, but objects get all their properties printed out), so this made sense when it was `console.error`. But Winston loggers have different arguments: `logger.*(message, objectWithMoreInfoForFormatting)`. It so happens that this *sort* of works because an error’s text is in the `message` property, and Winston will concatenate any `message` property in the info object with the message argument. But I don’t think you meant to do that (and if you did, I think we should be more explicit about it; this code is leveraging some serious subtleties).